src/components/SpecialResolution/SigningParty.vue

```diff
@@ -1,14 +1,17 @@
 
 <template>
   <div id="special-resolution-signing-party">
-    <!-- Resolution Signature, can make this more generic later. -->
-    <header id="resolution-signature-info-header">
-      <h2>Resolution Signature</h2>
-    </header>
-
-    <p class="section-description mt-2">
-      Enter the full name of the person who signed the special resolution and the date they signed it.
-    </p>
+    <!-- Hide if it's a correction. -->
+    <div v-if="isSpecialResolutionFiling && isEditing">
```

Comment:
I think it would be better to say "show only if ..."

Reply:
Figured it's not entirely clear this component could be used for corrections - which is why I put it in there

Reply:
OK, now I'm really confused. I don't mind more detailed comments if they explain things.

Reply:
I see you removed the comments. I guess that's OK -- the future developer can read the logic and try to figure things out. They won't be misled by comments.